files/.github/workflows/semantic-pull-request.yml

```diff
@@ -0,0 +1,27 @@
+name: "Semantic PR"
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - synchronize
+
+jobs:
+  main:
+    name: Validate PR title
+    runs-on: ubuntu-latest
+    steps:
+      - uses: amannn/action-semantic-pull-request@01d5fd8a8ebb9aafe902c40c53f0f4744f7381eb # v5.0.2
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          # Configure which types are allowed (newline delimited).
+          types: |
+            feat
+            fix
+            chore
+            docs
+            deps
+            test
```

Comment:
We also allow `refactor` and `ci`. Something to consider :)

Reply:
I added those two here.